build_docs.py

```diff
@@ -596,8 +540,7 @@ def parse_args():
     parser.add_argument(
         "-b",
         "--branch",
-        choices=dict.fromkeys(chain(*((v.branch_or_tag, v.name) for v in VERSIONS))),
-        metavar=Version.current_dev().name,
+        metavar="3.7",
```

Comment:
```suggestion
        metavar="3.12",
```
nit: maybe suggest the latest stable and supported version?

Reply:
Here you go :)